src/visitors/rules/structural/validateOpenAPISchema.js

```diff
@@ -10,157 +10,240 @@ class ValidateOpenAPISchema {
+          }
         }
         return null;
       },
       maxItems(node, ctx) {
         if (node && node.maxItems) {
-          if (typeof node.maxItems !== 'number') return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('number'), 'value');
-          if (node.maxItems < 0) return ctx.createError('Value of maxItems must be greater or equal to zero. You can`t have negative amount of something.', 'value');
+          if (typeof node.maxItems !== 'number') {
+            return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('number'), 'value');
+          }
+          if (node.maxItems < 0) {
+            return ctx.createError(
+              'Value of maxItems must be greater or equal to zero. You can`t have negative amount of something.',
+              'value',
+            );
+          }
         }
         return null;
       },
       minItems(node, ctx) {
         if (node && node.minItems) {
-          if (typeof node.minItems !== 'number') return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('number'), 'value');
-          if (node.minItems < 0) return ctx.createError('Value of minItems must be greater or equal to zero. You can`t have negative amount of something.', 'value');
+          if (typeof node.minItems !== 'number') {
+            return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('number'), 'value');
+          }
+          if (node.minItems < 0) {
+            return ctx.createError(
+              'Value of minItems must be greater or equal to zero. You can`t have negative amount of something.',
+              'value',
+            );
+          }
         }
         return null;
       },
       uniqueItems(node, ctx) {
         if (node && node.uniqueItems) {
-          if (typeof node.uniqueItems !== 'boolean') return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('boolean'), 'value');
+          if (typeof node.uniqueItems !== 'boolean') {
+            return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('boolean'), 'value');
+          }
         }
         return null;
       },
       maxProperties(node, ctx) {
         if (node && node.maxProperties) {
-          if (typeof node.maxProperties !== 'number') return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('number'), 'value');
-          if (node.maxProperties < 0) return ctx.createError('Value of maxProperties must be greater or equal to zero. You can`t have negative amount of something.', 'value');
+          if (typeof node.maxProperties !== 'number') {
+            return ctx.createError(ctx.messageHelpers.fieldTypeMismatchMessageHelper('number'), 'value');
+          }
+          if (node.maxProperties < 0) {
+            return ctx.createError(
+              'Value of maxProperties must be greater or equal to zero. You can`t have negative amount of something.',
```

Comment:
```suggestion
              'Value of maxProperties must be greater than or equal to zero.',
```

I'm removing this because it doesn't add any information.